coala_quickstart/coala_quickstart.py

```diff
@@ -50,17 +50,20 @@ def main():
     log_printer = LogPrinter(printer)
 
     project_dir = os.getcwd()
+    noninteractive = True
```

Comment:
It shouldn't be non-interactive by default. It's interactive by default (that is, `noninteractive = False`). Anyway, see the  otehr comment - there's no need for this variable.